packages/zoe/src/internal-types.js

```diff
@@ -96,14 +96,17 @@
  * @typedef {Object} InstanceAdmin
  * @property {() => void} assertAcceptingOffers
  * @property {(invitationHandle: InvitationHandle,
-      initialAllocation: Allocation,
-      proposal: ProposalRecord) => UserSeat } makeUserSeat
+ *     initialAllocation: Allocation,
+ *     proposal: ProposalRecord) => Promise<UserSeat> } makeUserSeat
+ * @property {(initialAllocation: Allocation,
+ *     proposal: ProposalRecord,
+ *     exitObj: ExitObj,
+ *     seatHandle: SeatHandle) => foo } makeNoEscrowSeat
```

Comment:
The `foo` here looks like something that got accidentally added. Can we change this to:

```suggestion
 * @property {MakeNoEscrowSeat} makeNoEscrowSeat
```

Thank you for catching that this was missing!